content/docs/studio/user-guide/projects-and-experiments/live-metrics.md

```diff
@@ -0,0 +1,104 @@
+# Generate live (real-time) metrics for experiments
+
+In your model training CI action, you can use [DVCLive] to send live updates to
+metrics and plots back to Iterative Studio, without writing them to your Git
+repository. This requires a 3-step process:
+
+1. [Set up an access token](#set-up-an-access-token)
+2. [Configure your model training CI job](#configure-your-model-training-ci-job)
+3. [Send and view the updates](#send-the-updates)
+
+Live metrics must be sent to this ingestion end-point: TODO: Add the end-point
+here
+
+https://www.youtube.com/watch?v=hKf4twg832g
+
+TODO: Replace this with a video tutorial for live metrics
+
+## Set up an access token
+
+Iterative Studio uses access tokens to authorize [DVCLive] to send live updates
+to the metrics and plots.
+
+### Generate access token
+
+To set up the access token, open your user profile page. In the
+`Studio access token` section, click on `Generate new token`.
+
+### Regenerate access token
+
+You can regenerate the access token at any point. The old access token will no
+longer be authorized to send live metrics and plots to Iterative Studio.
+
+### Reset (delete) access token
+
+You can delete the access token when you no longer need it.
+
+The option to delete the access token is also available when you try to change
+your password. That is, you can reset all your access credentials (your password
+and the access token) at once. This is handy if you suspect that your account
+security may have been compromised.
+
+## Configure your model training CI job
+
+You should define the following environment variables in your CI job:
+
+- `STUDIO_ENDPOINT`: The Iterative Studio ingestion endpoint. Set this to
+  `https://studio.iterative.ai/api/live`
```

Comment:
End users don't need to set this env var, the default value already is `https://studio.iterative.ai/api/live`. The env var was added for internal usage (i.e. post to local server)

```suggestion
```

Reply:
Thanks @daavoo